quicksort/quick_sort.py

```diff
@@ -62,6 +62,7 @@ def main():
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
+    
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/quicksort/quick_sort.py
+++ b/quicksort/quick_sort.py
@@ -62,7 +62,7 @@
     a = [1, 2, 1, 2, 3, 1, 2, 2, 1]
     quick_sort(a, 0, len(a) - 1)
     print(a)
-    
+
 
 if __name__ == '__main__':
     main()
```